guides/common/modules/proc_installing-postgresql.adoc

```diff
@@ -2,8 +2,7 @@
 = Installing PostgreSQL
 
 You can install only the same version of PostgreSQL that is installed with the `{foreman-installer}` tool during an internal database installation.
-You can install PostgreSQL using {RHELServer} 7 repositories or from an external source, as long as the version is supported.
-{Project} supports PostgreSQL version 12.1.
+{Project} supports PostgreSQL version 12.
```

Comment:
There's another occurrence of `PostgreSQL version 12.1` in another file (under considerations)